pir/pir-internal/src/main/java/com/duckduckgo/pir/internal/scan/PirScan.kt

```diff
@@ -94,10 +93,15 @@ class RealPirScan @Inject constructor(
         lastName = "Smith",
         city = "Chicago",
         state = "IL",
-        addresses = listOf(),
+        addresses = listOf(
```

Comment:
This is needed to get proper extracted profile matches (not documented anywhere)